src/index.js

```diff
@@ -154,7 +153,21 @@ export default class Headroom extends Component {
   }
 
   render () {
-    let style = {
+    const divProps = Object.assign(this.props)
+    delete divProps.onUnpin
+    delete divProps.onPin
+    delete divProps.onUnfix
+    delete divProps.disableInlineStyles
+    delete divProps.disable
+    delete divProps.parent
+    delete divProps.children
+    delete divProps.upTolerance
+    delete divProps.downTolerance
+    delete divProps.pinStart
+
+    const { style, wrapperStyle, ...rest } = divProps
```

Comment:
I was wondering... could you shorten this by doing something like this?

e.g. (untested):

```
    const { style, wrapperStyle, ...rest } = this.props
    const redundantKeys = Object.keys(this.propTypes)
    redundantKeys.forEach((key) => {
        delete rest[key]
    })
```